ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -96,44 +104,41 @@ export class Mint extends IronfishCommand {
     }
 
     const status = await client.getNodeStatus()
```

Comment:
this can move into the offline if statement, so we don't need to do this request.